src/test/java/com/google/api/generator/engine/ast/ClassDefinitionTest.java

```diff
@@ -23,8 +23,19 @@
 public class ClassDefinitionTest {
 
   @Test
-  public void validClassDefinition_basic() {
+  public void validClassDefinition_basicWithComments() {
+    LineComment lineComment = LineComment.withComment("AUTO-GENERATED DOCUMENTATION AND CLASS");
+    JavaDocComment javaDocComment =
+        JavaDocComment.builder()
+            .addComment("Base stub class for Google Example Library API.")
+            .addParagraph(
+                "This class is for advanced usage and reflects the underlying API directly.")
+            .build();
     ClassDefinition.builder()
+        .setCommentStatements(
+            Arrays.asList(
+                CommentStatement.withComment(lineComment),
+                CommentStatement.withComment(javaDocComment)))
         .setPackageString("com.google.example.library.v1.stub")
```

Comment:
Since the comment logic is not complicated, so I integrate the unit test w/ the existing test, same as JavaWriterVisitor test. (The place to print the comment matters, it should be after imports and before annotations/class scope). LMK if it makes to you @miraleung  Thanks

Reply:
SGTM.